crm_lead_firstname/readme/INSTALL.rst

```diff
@@ -0,0 +1,8 @@
+Since leads are expected to create partners only when needed and after
```

Comment:
I'd move this paragraph to DESCRIPTION.rst

Reply:
@ntsirintanis you forgot to remove it here ;)

Reply:
Damn, that's what happens when you forget to ctrl + s after editing! Thanks